### PR TITLE
Fixes Android: no double play and no canOpenUrl

### PR DESCRIPTION
**Pull request by tpreusse**
I was able to reproduce the double play issue on Android in my emulator and managed to fix it there.

And I've created a patch to skip `canOpenURL` in the webview. Make sure to re-run `yarn install` or the postinstall script.

### CODE REVIEW

**Comment by ovbm** [comment]
awesome! will test after lunch. How did you figure out it was related to the audioId?

File: src/components/AudioPlayer/index.js [hunk 1]
```diff
@@ -112,11 +120,12 @@ const AudioPlayer = () => {
         await TrackPlayer.reset()
         return
       }
+      const shouldAutoPlay = getAudioId(autoPlayAudio) === audioId
```
Comment (ovbm) [inline src/components/AudioPlayer/index.js:123]: I don't undertsand what's happening here. autoPlayAudio is boolean and always true, when the audioplayer is opened via message api.

**Comment by tpreusse** [comment]
I debugged with `console.log` and realized that it called `TrackPlayer.add` twice in quick succession (maybe 1ms). `TrackPlayer.getCurrentTrack`, `TrackPlayer.reset` and `TrackPlayer.add` seem to be not race condition safe on android. Which then lead down a path with more stable `audioId`s instead of auto play boolean etc.

File: src/components/AudioPlayer/index.js (hunk 1 above)
Comment (tpreusse) [reply src/components/AudioPlayer/index.js:123]: `autoPlayAudio` is no longer a boolean but a full audio object that should be auto played.

**Pull request merged**